From 3671bf3903964b21916dc30f6e3ee60692cb744c Mon Sep 17 00:00:00 2001
From: Enis Inan <enis.inan@enis.inan-C02M48DGFD57>
Date: Thu, 5 Oct 2017 14:08:03 -0700
Subject: [PATCH] (patch) AIX Ruby libpath with opt dir

Prior to this commit, build of Ruby on AIX with the --with-opt-dir
configure flag fails due to whitespace in the -blibpath arguments
arguments passed to the linker. (This patch uses a fragment derived
from the "non-portable sed" patch we used for Ruby 1.9.3.)
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index c69cb40..c0eb9b1 100644
--- a/configure.in
+++ b/configure.in
@@ -3353,7 +3353,7 @@ if test "x$OPT_DIR" != x; then
         for dir in $OPT_DIR; do
             echo x ${LIBPATHFLAG} ${RPATHFLAG} |
             sed "s/^x *//;s${IFS}"'%1\\$-s'"${IFS}${dir}/lib${IFS}g;s${IFS}%s${IFS}${dir}/lib${IFS}g"
-        done | tr '\012' ' ' | sed 's/ *$//'`
+        done | tr '\012' ' ' | sed "s% \+% %g;s% /%${IFS}/%g"`
     if test x"$val" != x; then
 	test x"${LDFLAGS}" = x || LDFLAGS="$LDFLAGS "
 	LDFLAGS="$LDFLAGS$val"
-- 
2.13.5 (Apple Git-94)

